optaplanner-docs/src/main/asciidoc/ConstraintStreams/ConstraintStreams-chapter.adoc

```diff
@@ -425,15 +425,19 @@ There are several collectors available out of the box. You can also provide your
 [collectorsOutOfTheBox]
 ==== Out-of-the-box collectors
 
-The following section focuses on the collector implementations provided out of the box. This section only describes the
-`int`-based variants of the collectors in detail. Many of the collectors also provide variants for other applicable
-result data types, such as `long`, `BigDecimal` or `Duration`. You can find a complete list by exploring the
-`org.optaplanner.core.api.score.stream.ConstraintCollectors` class.
```

Comment:
The docs should list all build-in collector methods for convenience. I believe this change now does that.